src/main/java/com/google/devtools/build/lib/buildeventservice/BuildEventServiceOptions.java

```diff
@@ -88,13 +88,14 @@
   public boolean besLifecycleEvents;
 
   @Option(
-    name = "project_id",
+    name = "bes_instance_name",
+    oldName = "project_id",
     defaultValue = "null",
     documentationCategory = OptionDocumentationCategory.LOGGING,
     effectTags = {OptionEffectTag.AFFECTS_OUTPUTS},
     help = "Specifies the BES project identifier. Defaults to null."
```

Comment:
Can we update the help string, eg. "Specifies the instance name under which BES will store uploaded BEP. Defaults to null."